### ISSUE TITLE
Tomb Raider 1996 steam + Tomb1Main

### ISSUE THREAD

**Issue by WildKing-droid** [legluondunet/MyLittleLutrisScripts#151]
Hello, i have this weird problem that lutris can't recognize/find p7zip on my machine. I'm sure that it is installed, i checked and reinstalled it, but lutris still is giving me something like this. I'm using Nobara Linux and lutris is installed with the distro by default. Any help or tips would be appreciated.
![Screenshot from 2023-07-07 11-45-14](https://github.com/legluondunet/MyLittleLutrisScripts/assets/84460621/c8188b84-413c-467f-b370-a95ee16852e8)

**Comment by legluondunet** [comment]
Hello @WildKing-droid , thank you for your report.
How did you install Lutris? Flatpak package ?

**Comment by WildKing-droid** [comment]
Ye, it's probably flatpak, Lutris came with Nobara distro.

**Comment by legluondunet** [comment]
I can reproduce this issue with Lutris Flatpak version, it can not find 7zip.
```

Started initial process 1014 from /usr/bin/bash -c cd "/home/legluondunet/Games/tomb-raider-i/drive_c/tmp"; ./bin2iso game.cue -c GAME.GOG ;./bin2iso game.cue; if [ -f "GAME-01.iso" ]; then cd=GAME-01.iso; else  cd=GAME.GOG; fi; echo $cd; ./7z x -y $cd;
Start monitoring process.

Jul 29 2017, 23:16:38
bin2iso V1.9b - Converts RAW format (.bin) files to ISO/WAV format
               Bob Doiron, ICQ#280251                     

Check for updates at http://users.andara.com/~doiron


FILE GAME.GOG BINARY
  TRACK 01 MODE1/2352
    INDEX 01 00:00:00

Jul 29 2017, 23:16:38
bin2iso V1.9b - Converts RAW format (.bin) files to ISO/WAV format
               Bob Doiron, ICQ#280251                     

Check for updates at http://users.andara.com/~doiron


Note: Appending pregap data to end of audio tracks

GAME-01.iso (234 Mb) - sectors 000000:120033 (offset 000000000:282319967)


Note: PreGap = 150 frames
Creating ./GAME-01.iso (000000,120033) Mode1/2048 :       000000001024002048003072004096005120006144007168008192009216010240011264012288013312014336015360016384017408018432019456020480021504022528023552024576025600026624027648028672029696030720031744032768033792034816035840036864037888038912039936040960041984043008044032045056046080047104048128049152050176051200052224053248054272055296056320057344058368059392060416061440062464063488064512065536066560067584068608069632070656071680072704073728074752075776076800077824078848079872080896081920082944083968084992086016087040088064089088090112091136092160093184094208095232096256097280098304099328100352101376102400103424104448105472106496107520108544109568110592111616112640113664114688115712116736117760118784119808Complete
GAME-01.iso
./7z: ligne 2: /usr/lib/p7zip/7z: Aucun fichier ou dossier de ce type
Monitored process exited.
Initial process has exited (return code: 32512)
All processes have quit
Exit with return code 32512
```

**Comment by legluondunet** [comment]
Fixed with commit 1d354fd
